File: Cap 03/Topico 3.9/Adicao de Numeros/adicnum.cs
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: Programacao - Topico Programacao.9
 * 
 * Programa ..: ADICAO DE DOIS NÚMEROS INTEIROS
*/

using System;

class Program
{
  static void Main()
  {
	  
    int X;
    int A;
    int B;
    
    Console.WriteLine("ADICAO DE NUMEROS");
    Console.WriteLine();
    
    Console.Write("Entre o 1o. valor numerico inteiro: ");
    A = Convert.ToInt32(Console.ReadLine());
    
    Console.Write("Entre o 2o. valor numerico inteiro: ");
    B = Convert.ToInt32(Console.ReadLine());
    
    X = A + B;
    
    Console.WriteLine("Resultado da adicao = " + X);
    
  }
  
}

File: Cap 03/Topico 3.9/Adicao de Numeros/adicnum.go
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: Programacao - Topico Programacao.9
 * 
 * Programa ..: ADICAO DE DOIS NÚMEROS INTEIROS
 */

package main

import (
  "fmt"
)

var X int
var A int
var B int

func main() {
	
  fmt.Println("ADICAO DE NUMEROS")
  fmt.Println()
  fmt.Print("Entre o 1o. valor numerico inteiro: ")
  fmt.Scan(&A)
  fmt.Print("Entre o 2o. valor numerico inteiro: ")
  fmt.Scan(&B)
  X = A + B
  fmt.Println("Resultado da adicao =", X)
  
}
